### PR TITLE
feat(caseOverview) group by documents name

### PR DESCRIPTION
**Pull request by rbioteau**
* Group files by document name
* Only display the latest version of each document
* Use FileViewer widget to download document

Related to BPO-74

### CODE REVIEW

File: uid-pages/form-case-overview/src/caseoverview.json [hunk 1]
```diff
@@ -742,7 +742,7 @@
...
...values(object){\n    if(!object){\n        return [];\n    }\n    return  Object.keys(object).map(function(key){\n        return $data.context[key];\n    });    \n}\n\nvar businessDataRefs =  values($data.context).filter(function(ref){\n        return ref && ref.type;\n    }).map(function(businessDataRef){\n        return {\n          \"name\" : businessDataRef.name + \" (\"+ businessDataRef.type +\")\",\n          \"link\" : businessDataRef.link\n        };\n    });\n\nvar documentRefs = $data.openedContext ? $data.openedCaseDocuments : $data.archivedDocuments;\n\nvar groupBy = function(xs, key) {\n  return xs.reduce(function(rv, x) {\n    (rv[x[key]] = rv[x[key]] || []).push(x);\n    return rv;\n  }, {});\n};\n\nconsole.log(groupBy(documentRefs,\"name\"));\n\nreturn {\n    \"found\" : !!$data.case,\n    \"id\" : $data.case.id,\n    \"sourceObjectId\" : $data.case.rootCaseId ?  $data.case.rootCaseId  : $data.case.id ,\n    \"processName\" : $data.case.processDefinitionId.displayName + \" (\"+ $data.case.processDefinitionId.version+\")\",\n    \"businessDataRefs\" : businessDataRefs,\n    \"selectedBusinessDataRef\" : businessDataRefs[0],\n    \"documentRefs\" : documentRefs \n};",
```
Comment (abirembaut) [inline uid-pages/form-case-overview/src/caseoverview.json]: I guess the console.log should be removed ?

Comment (rbioteau) [reply uid-pages/form-case-overview/src/caseoverview.json]: yep

**Comment by abirembaut** [comment]
Good Job ! It works well. But do we really want to also display the content type ? It seems too technical to me...

**Comment by rbioteau** [comment]
Ask to a PO I guess.

**Pull request merged**